modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -600,6 +600,32 @@
           end
         end
 
+        context 'when the claim is complete' do
+          let(:bgs_claim) { build(:bgs_response_with_lc_status).to_h }
+
+          it 'shows a closed date' do
+            with_okta_user(scopes) do |auth_header|
+              VCR.use_cassette('bgs/tracked_items/find_tracked_items') do
+                VCR.use_cassette('evss/documents/get_claim_documents') do
+                  expect_any_instance_of(BGS::EbenefitsBenefitClaimsStatus)
+                    .to receive(:find_benefit_claim_details_by_benefit_claim_id).and_return(bgs_claim)
+                  expect(ClaimsApi::AutoEstablishedClaim)
+                    .to receive(:get_by_id_and_icn).and_return(nil)
+
+                  get claim_by_id_path, headers: auth_header
```

Comment:
Just to confirm my understanding, the VCR cassettes return previously-recorded external API responses, which are then used for an actual call to the get claim by ID endpoint?

Reply:
yes, the recorded responses are used when running the tests. [Here](https://www.rubyguides.com/2018/12/ruby-vcr-gem/) is a reference.